react/features/face-centering/functions.js

```diff
@@ -0,0 +1,107 @@
+// @flow
+
+import { getBaseUrl } from '../base/util';
+
+import { FACE_BOX_EVENT_TYPE, DETECT_FACE_BOX } from './constants';
+import logger from './logger';
+
+/**
+ * Sends the face box to all the other participants.
+ *
+ * @param {Object} conference - The current conference.
+ * @param  {Object} faceBox - Face box to be sent.
+ * @returns {void}
+ */
+export function sendFaceBoxToParticipants(
+        conference: Object,
```

Comment:
Drop the flow stuff from the whole file please

Reply:
cant, eslint will crash, talked with @saghul  about this